packages/forms-web-app/src/views/check-answers.njk

```diff
@@ -371,7 +371,7 @@
         ]
       }) }}
 
-      <form action="submission" method="get" novalidate>
+      <form action="/appellant-submission/submission" method="get" novalidate>
```

Comment:
this is definitely not right, but is something to be covered in a future ticket already